src/main/java/uk/gov/hmcts/juror/api/moj/controller/reports/request/JurySummoningMonitorReportRequest.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.hmcts.juror.api.moj.controller.reports.request;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Pattern;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.hmcts.juror.api.moj.validation.dto.ConditionalDtoValidation;
+import uk.gov.hmcts.juror.api.validation.ValidateIf;
+import uk.gov.hmcts.juror.api.validation.ValidateIfTrigger;
+import uk.gov.hmcts.juror.api.validation.ValidationConstants;
+
+import java.time.LocalDate;
+import java.util.List;
+
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@ConditionalDtoValidation(
+    conditionalProperty = "searchBy", values = {"POOL"},
+    requiredProperties = {"poolNumber"},
+    message = "poolNumber is required for POOL search")
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+@ValidateIfTrigger(classToValidate = JurySummoningMonitorReportRequest.class)
+public class JurySummoningMonitorReportRequest {
+
+    @Pattern(regexp = "^(POOL|COURT)$")
+    @NotBlank
+    private String searchBy;
```

Comment:
Would be better as a enum

Reply:
using strings as it works with the conditional validation annotation